File: c#/program2.cs
using System;

class Arithmetic
{
    public int No1;
    public int No2;

    public Arithmetic(int no1, int no2)
    {
        this.No1 = no1;
        this.No2 = no2;
    }

    public int Addition()
    {
        int Ans = 0;
        Ans = this.No1 + this.No2;
        return Ans;
    }
}

class program2
{
    public static void Main(string[] args)
    {
        int iValue1 = 0;
        int iValue2 = 0;
        int iRet = 0;

        Console.WriteLine("Enter first number");
        iValue1 = Convert.ToInt32(Console.ReadLine());

        Console.WriteLine("Enter second number");
        iValue2 = int.Parse(Console.ReadLine());

        Arithmetic obj = new Arithmetic(iValue1,iValue2);
        iRet = obj.Addition();

        Console.WriteLine("Addition is : "+iRet);
    }
}

File: scala/program2.scala
import scala.io.StdIn.{readInt}

class Addition(var no1 : Int,var no2 : Int)
{
    def Arithmatic(): Int = {
        var ans = this.no1 + this.no2
        return ans;
    }
}

object Demo
{
    def main(args : Array[String])
    {
        println("Enter first number.");
        var no1 : Int = readInt()
        println("Enter second number.")
        var no2 : Int = readInt()

        var obj = new Addition(no1,no2);
        var res = obj.Arithmatic();

        println("Output : "+res);
    }
}